### DISCUSSION TITLE
无法build 依赖于xrepo仓库fmt包的简单程序

### DISCUSSION THREAD

**Discussion by Ramblestsad** [xmake-io/xmake-vscode#122] [Q&A] [answered]
<img width="441" alt="image" src="https://user-images.githubusercontent.com/45003009/176390199-b20860c4-c865-4a05-8218-cf32cf7a108b.png">
<img width="414" alt="image" src="https://user-images.githubusercontent.com/45003009/176390243-83c868ad-edd9-4a4c-9734-b0ea2f1740cb.png">
<img width="1277" alt="image" src="https://user-images.githubusercontent.com/45003009/176390363-edf95ec0-e886-4d13-adae-78c8b3be1562.png">
<img width="1143" alt="image" src="https://user-images.githubusercontent.com/45003009/176390525-a2085a18-082c-402f-9e81-eb4d729eb9c8.png">

这是为啥呀？

**Comment by waruqi** [comment, marked as answer]
set_languages("c++14")
